yarn.lock

```diff
@@ -16541,7 +16576,7 @@ xss@1.0.10:
     commander "^2.20.3"
     cssfilter "0.0.10"
 
-"xstate5@npm:xstate@5":
+"xstate5@npm:xstate@5", xstate@^5.13.0:
```

Comment:
Please remove such changes to xState. After this, it will LGTM

Reply:
@mastro993 I am right?

Reply:
@mastro993 @Vangaorth Currently, if I remove this line, `yarn` installs two different versions the next time it runs: `5.13.0` and `5.17.4`. If we don't make any changes, there is only one version: `5.13.0`.
I'd leave things as they are.

Reply:
@dmnplb Before such change we must test if the CI builds works correctly and XState machines have the expected behavior.